erdgo/storage/mapCacher.go

```diff
@@ -122,6 +124,24 @@ func (mc *mapCacher) Len() int {
 	return len(mc.dataMap)
 }
 
+// SizeInBytesContained returns the size in bytes of all contained elements
+func (mc *mapCacher) SizeInBytesContained() uint64 {
+	mc.RLock()
+	defer mc.RUnlock()
+
+	total := 0
+	b := new(bytes.Buffer)
+	for _, v := range mc.dataMap {
+		if err := gob.NewEncoder(b).Encode(v); err != nil {
```

Comment:
could have split this in 2 lines. 
What happens if the encode method errors? The code will silently continue returning an **incorrect value**. Is this desirable? Why not returning the error also?